frontend/src/scenes/insights/InsightTabs/InsightDisplayConfig.tsx

```diff
@@ -94,7 +96,7 @@ export function InsightDisplayConfig({
     return (
         <div className="display-config-inner">
             <div className="display-config-inner-row">
-                {showDateFilter[activeView] && (
+                {showDateFilter[activeView] && !disableTable && (
```

Comment:
I think it's kinda weird that disableTable disables the date range thingy, but I guess it's fine for now, will come and clean this up once the datefilters are fixed!